assets/src/edit-story/components/form/dateTime/dateTime.js

```diff
@@ -40,13 +45,19 @@ const DateTimeWrapper = styled.div`
   padding: 4px;
 `;
 
+const StyledButton = styled(Button)`
+  color: rgb(0, 117, 175);
```

Comment:
Hm, this is a colour from WP calendar, since we're not using it anyway, will look for a better colour until we redesign the calendar.